README.rst

```diff
@@ -13,3 +13,131 @@ __ https://discuss.openedx.org/c/working-groups/build-test-release/30
 More details about the working group are at the `BTR page in the wiki`__.
 
 __ https://openedx.atlassian.net/wiki/spaces/COMM/pages/1022099494/Build+-+Test+-+Release+Working+Group
+
+================================
+Role definitions and assignments
+================================
+
+*Current release: Lilac*
+
+The following roles, with corresponding expectations and assignments, are
+defined for the current release cycle.
+
+Group Chair
+===========
+
+*Current assignee: @arbrandes*
+
+The group chair is responsible for coordinating efforts that culminate in
+getting a new release out and maintaining the current one.  In particular, the
+Group Chair will:
+
+* Manage the current release cycle in general, including making sure that
+  proper deadlines are established and adhered to
```

Comment:
Punctuation at the end of the bullet sentences would make this more readable.

Reply:
I'm never sure whether to use punctuation at the send of bullet points or not, so from now on I'll just adopt a "yes".  :)